src/SectionPage.tsx

```diff
@@ -0,0 +1,38 @@
+import './App.css';
+import './css_reset.css'
+import './section_view.css';
+import * as types from './types';
+import {Files} from './Files';
+import {ChordProgression} from './ChordProgression';
+import {Comments} from './Comments';
+import {CreateComment} from './CreateComment';
+import {SectionTitle} from './SectionTitle';
+import {useGlobalStore} from './hooks/useGlobalStore';
+
+type SectionPageProps = {
+    projectId: string;
+    sectionId: string;
+}
+
+const SectionPage: React.FC<SectionPageProps> = ({projectId, sectionId}) => {
+    const globalStore = useGlobalStore();
+    const section = globalStore.getSection(sectionId);
+    const files = globalStore.getFilesForSection(sectionId);
+
+    const sectionPointer: types.EntityPointer = {
+        entityId: sectionId,
+        entityType: types.EntityType.SECTION,
+    };
+
+    return (
+        <div className="root">
+            <SectionTitle sectionId={sectionId} />
+            <ChordProgression chordProgression={section.chordProgression} />
```

Comment:
I made a lot of whitespace changes in the PR. Please let me know what you think about:

- 4 spaces indent instead of 2 spaces - I personally feel strongly about this. It helps me scan the code much quicker and efficiently this way. The one argument someone may have against it is that it can make the code look too indented. My argument against that is that could be a sign that you need to address the number of levels of indentation happening on that line of code
- Named imports have no spacings inside the braces - I feel less strongly about this, but still prefer it

Reply:
I agree, I think four spaces is a great standard. Greatly enhances readability, which always goes a long way, even if that's not a measurable thing

Reply:
Swag